kfs-core/src/main/resources/edu/arizona/kfs/coa/document/datadictionary/FACostSubCategoryMaintenanceDocument.xml

```diff
@@ -0,0 +1,55 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:p="http://www.springframework.org/schema/p"
+    xmlns:dd="http://rice.kuali.org/dd"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+        http://rice.kuali.org/dd
+        http://rice.kuali.org/dd/dd.xsd">
+
+    <bean id="FACostSubCategoryMaintenanceDocument" parent="FACostSubCategoryMaintenanceDocument-parentBean" />
+  
+    <bean id="FACostSubCategoryMaintenanceDocument-parentBean" abstract="true" parent="MaintenanceDocumentEntry">
+       <property name="businessObjectClass" value="edu.arizona.kfs.coa.businessobject.FACostSubCategory" />
+       <property name="maintainableClass" value="org.kuali.rice.kns.maintenance.KualiMaintainableImpl" />
```

Comment:
The maintainable class should be:
property name="maintainableClass" value="org.kuali.kfs.sys.document.FinancialSystemMaintainable"